heimdall-middleware-spec/src/test/java/br/com/conductor/heimdall/middleware/util/helpermock/call/RequestMockTest.java

```diff
@@ -0,0 +1,79 @@
+package br.com.conductor.heimdall.middleware.util.helpermock.call;
+
+import br.com.conductor.heimdall.middleware.spec.Header;
+import br.com.conductor.heimdall.middleware.spec.Query;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import static org.junit.Assert.*;
+
+import java.util.HashMap;
+
+public class RequestMockTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+  @Rule public final Timeout globalTimeout = new Timeout(10000);
+
+  @Test
+  public void testHeaderOutputNotNull() {
+    final RequestMock objectUnderTest = new RequestMock();
+
+    final Header retval = objectUnderTest.header();
+
+    assertNotNull(retval);
```

Comment:
Don't use assertNotNull() if there is another assert that effectively checks the same thing.